yarn-project/foundation/src/abi/abi.ts

```diff
@@ -355,10 +365,13 @@ export function getFunctionDebugMetadata(
   functionArtifact: FunctionArtifact,
 ): FunctionDebugMetadata | undefined {
   if (functionArtifact.debugSymbols && contractArtifact.fileMap) {
-    const debugSymbols = JSON.parse(
+    const programDebugSymbols = JSON.parse(
       inflate(Buffer.from(functionArtifact.debugSymbols, 'base64'), { to: 'string', raw: true }),
     );
-    return { debugSymbols, files: contractArtifact.fileMap };
+    // TODO(https://github.com/AztecProtocol/aztec-packages/issues/5813)
```

Comment:
@sirasistant Pinging you for awareness. Not sure if there is anyone else who would be better to ping as well. You can look into that issue for more information.